cypher-shell/src/main/java/org/neo4j/shell/prettyprint/PrettyPrinter.java

```diff
@@ -93,13 +93,13 @@ private String pathAsString(Path path) {
 
         path.iterator().forEachRemaining(segment -> {
             if (segment.relationship() != null) {
-                list.add(relationshipAsString(segment.relationship()));
+                list.add("-" + relationshipAsString(segment.relationship()) + "->");
```

Comment:
The direction is not always to the right.

This is visible in your initial example:

```
neo4j> MATCH p=(a:Person)-[:WORKS_AT]->(d)<-[:WORKS_AT]-(b:Person) WHERE a.name = "Daniel" RETURN p LIMIT 2;
p
(:Person {name: "Daniel"})-[:WORKS_AT]->(:Org {name: "GraphIT"})-[:WORKS_AT]->(:Person {name: "Gabriele"})
(:Person {name: "Daniel"})-[:WORKS_AT]->(:Org {name: "GraphIT"})-[:WORKS_AT]->(:Person {name: "Jerry"})
neo4j>
```

The direction is always from the person to the organization, but your resulting print is

daniel -> org -> gabriele

The second arrow should point to the left